azure-pipelines.yml

```diff
@@ -440,10 +378,99 @@ stages:
 
   - template: stage-with-burgr-notifications.yml@commonTemplates
     parameters:
-      burgrName: 'qa'
-      burgrType: 'qa'
-      stageName: 'qa'
-      stageDisplayName: 'QA:'
+      burgrName: 'qa_ut_analysis'
+      burgrType: 'qa_ut_analysis'
+      stageName: 'qa_ut_analysis'
+      stageDisplayName: 'QA (UTs, SC Analysis, Mend Scan)'
+      stageDependencies:
+        - build
+      jobs:
+        - job: uts_analysis_mend
+          displayName: 'Run UTs, SC analysis and Mend scan'
+          variables:
+            sonarCloudOrganization: 'sonarsource'
+            sonarCloudMsBuildProjectKey: 'sonarscanner-msbuild'
+            sonarCloudMsBuildProjectName: 'SonarScanner for .NET'
```

Comment:
I think these can be dropped (in the Build stage as well) as the values is used only in the `Code Analysis - Begin` task.